packages/apps/transfer/src/services/faucet/index.ts

```diff
@@ -60,28 +61,26 @@ export const fundExistingAccount = async (
     throw new Error('Failed to sign transaction');
   }
 
-  const signature2 = sign(command.cmd, keyPair);
+  const signature2 = sign(transaction.cmd, keyPair);
 
   if (signature2.sig === undefined) {
     throw new Error('Failed to sign transaction');
   }
 
-  transactionBuilder.addSignatures(
-    {
-      pubKey: FAUCET_PUBLIC_KEY,
-      sig: signature1.sig,
-    },
-    { pubKey: keyPair.publicKey, sig: signature2.sig },
-  );
-
   const apiHost = getKadenaConstantByNetwork('TESTNET').apiHost({
     networkId: NETWORK_ID,
     chainId,
   });
 
-  await transactionBuilder.send(apiHost);
+  transaction.sigs = [{ sig: signature1.sig }, { sig: signature2.sig }];
 
-  return await transactionBuilder.pollUntil(apiHost, {
-    onPoll,
-  });
+  const { submit, pollStatus } = getClient(apiHost);
```

Comment:
Can we make this work with WalletConnect? Take the network from the walletconnect settings, this will open up using this on the devnet as well.
Might be better to do in a next PR though